infrastructure/server/oidc_clients_fam.tf

```diff
@@ -11,8 +11,8 @@ resource "aws_cognito_user_pool_client" "fam_console_oidc_client" {
     "https://oidcdebuggersecure-3d5c3f-dev.apps.silver.devops.gov.bc.ca/"
   ]
   logout_urls = [
-    "${var.front_end_redirect_path}",
-    "http://localhost:5173"
+    "${var.frontend_logout_chain_url}${var.front_end_redirect_path}",
+    "${var.frontend_logout_chain_url}http://localhost:5173"
```

Comment:
Hmm. Is the localhost redirect going to make it into production? I think it is. Not sure how to make sure this only appears in dev and test. Any ideas?
I guess the way to do it is to put it into the Terragrunt files instead of in the infrastructure file. That way we can add the localhost to dev and test, but not production.

Reply:
Good catch!
Thinking.... maybe we don't even need localhost one...

Reply:
um... but in the callback_urls we also have localhost? 
<img width="732" alt="Screen Shot 2023-03-02 at 12 45 13 PM" src="https://user-images.githubusercontent.com/23131735/222547683-7ea2a020-cc4e-4e34-ad1d-5aa90f0d48d7.png">

Reply:
Yeah, I missed that. Is we leave it in that file, it will go to production. It's a security risk. Both the callback_urls and the logout_urls should not have localhost in production. One could argue that the logout_urls doesn't really matter so much from a security perspective, but the localhost one is definitely a security risk.

Reply:
Thanks for the explaination Conrad! I'll think a bit how to do this

Reply:
Should be able to move the whole "list" to the terragrunt file and then specify it individually per environment. The syntax is the same I believe.